keps/sig-storage/695-skip-permission-change/README.md

```diff
@@ -193,7 +250,7 @@ type PodSecurityContext struct {
 ### Scalability
 
 * **Will enabling / using this feature result in any new API calls?**
-  This feature should cause no new API calls.
+  This feature could trigger additional api call from kubelet to update PVC status after permission/ownership change is complete.
```

Comment:
It *will* trigger a single PVC.Status update call for every volume that supports fsGroup per pod with fsGroup set.